workshop 01 - Introduction to Quantum Computing/intro-to-qsharp.ipynb

```diff
@@ -5,7 +5,8 @@
    "metadata": {},
    "source": [
     "# Introduction\n",
-    "\n",
+    "---\n",
+    "<p><i>by Rohit Ashiwal - Microsoft Student Partner - rohit.ashiwal265@gmail.com</i></p>\n",
```

Comment:
I'd prefer something like, `Rohit Ashiwal <rohit.ashiwal265@gmail.com>`. Like the old school DCO. But this is a choice that we had to make :) We can further have some kind of README wherein we can explain the roles of our members with their preferred "work" positions :D

Reply:
Sure! We have to create some sort of guidelines, it is really a good idea!